recipes/prefsync/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.2" %}
+{% set sha256 = "66f0d3870b442e715de23b632e83f27f5971885b40bb3f6b211437138c554496" %}
+
+package:
+  name: prefsync
+  version: {{ version }}
+
+source:
+  fn: prefsync-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/prefsync/prefsync-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - prefsync
+
+  commands:
+    - python -m prefsync --help
+    - prefsync --help
+
+about:
+  home: https://github.com/asmeurer/prefsync
+  license: MIT
+  summary: |
+    A little tool to help synchronize Mac OS X plist files (used for
+    preferences for most Mac Apps) seamlessly, in a way that can be tracked by
+    git.
+  description: |
+    A little tool to help synchronize Mac OS X plist files (used for
+    preferences for most Mac Apps) seamlessly, in a way that can be tracked by
+    git.
```

Comment:
So, my understanding from what has been relayed to me is the `summary` should be a quick one liner. The `description` can be a bit longer.